Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -113,26 +103,26 @@ td.formGridTopDataRow
 {
   visibility: hidden;
   transition-property: visibility;
-  transition-duration: 0;
-  transition-delay: 0;
+  transition-duration: 0s;
+  transition-delay: 0s;
   transition-timing-function: linear;
-
   content: attr(aria-label);
   white-space: pre;
   display: inline-block;
   position: absolute;
-  left: 0;
-  top: 1.2em;
-  z-index: 1000;
-  padding: 0.25em;
-  border: 1px solid Black;
-  background-color: Lightyellow;
-  box-sizing: border-box;
+  /* popup spawns at the bottom right corner of the icon's hover highlighting */
+  top: calc(var(--default-icon-size) + var(--remotion-themed-spacing-half));
+  padding: var(--remotion-themed-spacing) calc(var(--remotion-themed-spacing) + var(--remotion-themed-spacing-half));
+  border: var(--remotion-themed-tooltip-border);
+  color: var(--color-tooltip-text);
+  background-color: var(--color-tooltip-background);
+  font-size: 90%;
```

Comment:
--font-tooltip (or --tooltip-font etc, please chose based on the naming rules we have)